sandbox/keycloak-provision/main.ts

```diff
@@ -140,14 +140,19 @@ async function main() {
   // Upsert Admin client
   await kc.createRealmAdminServiceAccount(AUTH_REALM_NAME, ADMIN_CLIENT_ID, ADMIN_CLIENT_SECRET);
 
-  const authUsers = msUsers.map(({ surname, givenName, mail, jobTitle }) => ({
-    username: mail,
-    email: mail,
-    firstName: givenName,
-    lastName: surname,
-    password: mail,
-    roles: jobTitle ? jobTitle.split(',').map((role) => role.trim()) : [],
-  }));
+  const authUsers = msUsers.map(
+    ({ surname, givenName, mail, jobTitle, extension_85cc52e9286540fcb1f97ed86114a0e5_bcgovGUID }) => ({
+      username: mail,
```

Comment:
what is the `extension_xxxx_bcgovGUID`? Wondering if it is something that should be pulled from a config file?